x/stakeibc/keeper/msg_server_redeem_stake.go

```diff
@@ -58,7 +58,7 @@ func (k msgServer) RedeemStake(goCtx context.Context, msg *types.MsgRedeemStake)
 	balance := k.bankKeeper.GetBalance(ctx, sender, coinDenom)
 	k.Logger(ctx).Info(fmt.Sprintf("Redemption issuer IBCDenom balance: %v%s", balance.Amount, balance.Denom))
 	k.Logger(ctx).Info(fmt.Sprintf("Redemption requested redemotion amount: %v%s", inCoin.Amount, inCoin.Denom))
-	if balance.Amount.LT(sdk.NewInt(msg.Amount)) {
+	if balance.Amount.LT(sdk.NewInt(cast.ToInt64(msg.Amount))) {
```

Comment:
flagging

Reply:
Addressed per new cast fork and using cast.To____E